editor/src/components/canvas/canvas-strategies/strategies/absolute-resize-bounding-box-strategy.tsx

```diff
@@ -263,6 +264,13 @@ export function absoluteResizeBoundingBoxStrategy(
                   'starting-metadata',
                 ),
                 queueTrueUpElement(childGroups.map(trueUpGroupElementChanged)),
+                setActiveFrames([
```

Comment:
selectedElement, newFrame, ..... hmm these are also available in pushIntendedBoundsAndUpdateGroups

the action: 'resize' is the one piece that's missing

actually I wonder if the name `pushIntendedBoundsAndUpdateGroups` could be changed to `setStrategyIntent`, so it could store resize, target, frame (even originalFrame). it feels like if a strategy forgets to call `pushIntendedBoundsAndUpdateGroups` that is a mistake. I wonder if `setStrategyIntent` could be elevated to the level of `StrategyApplicationResult`, meaning it would be impossible to return from the apply function without also setting what was the intent